### PR TITLE
Make it very clear that this repo is deprecated

### PR DESCRIPTION
**Pull request by gabebw**

### CODE REVIEW

**Comment by gabebw** [comment]
@danchoi @mxie

File: README.md [hunk 1]
```diff
@@ -1,44 +1,8 @@
...
-    heroku pg:psql --app railsbridge-boston
-    # wait for prompt
-    delete from completions;
-    delete from students;
-    \q 
-
-That should do it.
-
-
+This was the website for the RailsBridge Boston Workshop for Women and their
```
Comment (mxie) [inline README.md]: While you're in here, might as un-capitalize "Women".

**Comment by danchoi** [comment]
@mxie can we make @gabebw an admin of all the repos if he isn't already?

**Comment by mxie** [comment]
Yeah, I can do that.

**Comment by mxie** [comment]
@gabebw Feel free to squash and merge!

**Pull request merged**